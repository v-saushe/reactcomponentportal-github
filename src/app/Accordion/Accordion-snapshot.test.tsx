import React from 'react';
import { shallow } from 'enzyme'; 
import Acccordion from './Accordion';

test('renders the component', () => {
  const component = shallow(
  <Acccordion 
    title='From where I can login and avail services.'
   content='Currently theres no login facility available'
   />
   );
  expect(component).toMatchSnapshot();
});