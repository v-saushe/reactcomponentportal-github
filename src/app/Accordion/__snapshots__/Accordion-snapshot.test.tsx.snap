// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<Fragment>
  <div
    className="accordion__section"
  >
    <button
      className="accordion"
      onClick={[Function]}
    >
      <p
        className="accordion__title"
      >
        From where I can login and avail services.
      </p>
      <Chevron
        className="accordion__icon"
        fill="#777"
        width={10}
      />
    </button>
    <div
      className="accordion__content"
      style={
        Object {
          "maxHeight": "0px",
        }
      }
    >
      <div
        className="accordion__text"
        dangerouslySetInnerHTML={
          Object {
            "__html": "Currently theres no login facility available",
          }
        }
      />
    </div>
  </div>
</Fragment>
`;
